tools/deployments/validate-changesets.ts

```diff
@@ -35,6 +35,13 @@ export function validateChangesets(
 					);
 				}
 			}
+			if (
+				!/^feature:|feat:|fix:|refactor:|docs:|chore:/.test(changeset.summary)
```

Comment:
I preserved `feature:` and `feat:` as alternative spellings because we've used both extensively in the past, but happy to standardise on one if we think it's worth it